ladybug_vtk/_extend_sunpath.py

```diff
@@ -84,17 +94,21 @@ def sunpath_to_vtkjs(self, output_folder: str = '.', file_name: str = 'sunpath',
     minor_text_polydata = [to_text(text, compass.minor_azimuth_points[count].
                                    move(left_vector).move(down_vector), minor_scale)
                            for count, text in enumerate(compass.MINOR_TEXT)]
-    minor_label_dataset = ModelDataSet(
-        name='minor_labels', data=minor_text_polydata, color=Color())
+    minor_label_dataset = DisplayPolyData(
+        name='Sun path::Minor Labels', identifier='minor_labels',
```

Comment:
I didn't realize that we had support for 3D scene text in VTK. If so, then we should find a way to work this into the VisualizationSet since I imagine that a lot of people will want to use it in their graphics. I think we can probably add as it's own class that can be included in the ContextGeometry object. Does that sound good to you?